docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,502 @@
 .. highlight:: psql
 .. _recover:
+.. _conf-env:
 
-====================================
-Troubleshooting with Crate-Node CLI
-====================================
+===================================
+Troubleshooting with Crate node CLI
+===================================
+This part of the CrateDB documentation is based on the `Elasticsearch
+Reference [7.2] <https://www.elastic.co/guide/en/elasticsearch/reference/current/node-tool.html>`_.
 
-The crate-node CLI enables you to perform certain unsafe operations
-on a node that are only possible while it is shut down. This command allows you
-to adjust the role of a node and may be able to recover some data after a
-disaster.
+The Crate node CLI enables you to form a new cluster if a CrateDB cluster
```

Comment:
The purpose of the CLI is not just to form a new cluster. It is also used to change the cluster state of nodes, to be able to fix a broken cluster or nodes.